src/ray/util/logging.h

```diff
@@ -254,14 +258,25 @@ class RayLog {
   /// This function to judge whether current log is fatal or not.
   bool IsFatal() const;
 
+  /// TODO(hjiang): Having both log directory and filename for `StartRayLog` is confusing,
+  /// at the end of the day should have only filename. Fix all usage in the next PR.
+  ///
   /// The init function of ray log for a program which should be called only once.
   ///
   /// \parem appName The app name which starts the log.
   /// \param severity_threshold Logging threshold for the program.
-  /// \param logDir Logging output file name. If empty, the log won't output to file.
+  /// \param log_dir Logging output directory name.
+  /// \param log_filepath Logging output filepath.
+  ///
+  /// Both [log_dir] and [log_filepath] are used to determine log output filename; if
+  /// both empty, the log won't output to file, but to stdout. It's illegal to set
+  /// [log_filepath] and [log_dir] at the same time.
   static void StartRayLog(const std::string &appName,
                           RayLogLevel severity_threshold = RayLogLevel::INFO,
-                          const std::string &logDir = "");
+                          const std::string &log_dir = "",
+                          const std::string &log_filepath = "",
+                          size_t log_rotation_max_size = kDefaultLogRotationMaxSize,
```

Comment:
default_log_rotation_max_size

Reply:
google coding stlye suggests constant starts with `k` and CamelCase: https://google.github.io/styleguide/cppguide.html#Constant_Names

Reply:
I mean the parameter name. It's the default value when the env var is not set?

Reply:
I won't do that... otherwise you have to define a priority system for pass-ed value and env value.